### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by markus-marandi**

### CODE REVIEW

File: README.asciidoc [hunk 1]
```diff
@@ -62,12 +62,6 @@ Testing our quite limited codebase took more time than was expected beforehand a
 
 == Electronics
```
Comment (ReikoR) [inline README.asciidoc:63]: There are no photos of the mainboard.

File: README.asciidoc [hunk 2]
```diff
@@ -86,119 +80,20 @@ reverse voltage protection for avoiding reverse voltage damaging the
...
-mm. Would have been easier on the mainboard side to solder.
-* Cable management was difficult with Test robot – should have marked
-the cables to have better overview. But it turned out ok.
 
-* {blank}
-+
+=== A short analysis of what was good and bad in your electronics and how to improve.
 
+* The PCB should had a bit more room on the mainboard side for the better soldering 
+* 5V buck converter was very fragile and when even a slight current fluctuation happened, it burned down and also burned down the isolators
```
Comment (ReikoR) [inline README.asciidoc:95]: > 5V buck converter

5V converter is linear, not switching regulator.

**Review by ReikoR — CHANGES_REQUESTED** [review]

File: README.asciidoc (hunk 2 above)
Comment (markus-marandi) [reply README.asciidoc]: solved

File: README.asciidoc (hunk 1 above)
Comment (markus-marandi) [reply README.asciidoc:63]: added photo

**Comment by ReikoR** [comment]
Do not add `.DS_Store` files to repository.

**Comment by markus-marandi** [comment]
corrected gitignore

**Review by ReikoR — APPROVED** [review]

**Pull request merged**